.expeditor/third-party-packages.pipeline.yml

```diff
@@ -12,3 +12,19 @@ steps:
           host_os: windows
           environment:
 
+  - label: ":mac_os_x: :github: Create PR to update Homebrew/homebrew-cask"
+    command:
+      - bash .expeditor/create_pr_for_homebrew.sh
+    expeditor:
+      accounts:
+        - github/chef
+      secrets:
+        GITHUB_TOKEN:
```

Comment:
I don't see this being used in the script you wrote. Is it required by the `open_pull_request` function?

Reply:
Yes